lemmy_apub/src/fetcher.rs

```diff
@@ -258,15 +251,15 @@ pub(crate) async fn get_or_fetch_and_upsert_user(
         return Ok(u);
       }
 
-      let mut uf = UserForm::from_apub(
+      let user = User_::from_apub(
         &person?,
         context,
         Some(apub_id.to_owned()),
         recursion_counter,
       )
       .await?;
-      uf.last_refreshed_at = Some(naive_now());
-      let user = blocking(context.pool(), move |conn| User_::update(conn, u.id, &uf)).await??;
+      // TODO: do we need to set this? would need a separate db call
+      //uf.last_refreshed_at = Some(naive_now());
```

Comment:
Is this important?

Reply:
Yes, it needs to update that last refreshed at column after its been fetched.

Reply:
Ah yes of course we do. I will add a new method in the db for that.